### PR TITLE
Enable selenium grid support#40

### PR DESCRIPTION
**Pull request by s4int**

### CODE REVIEW

**Comment by kahunacohen** [comment]
Do we need a test for this?

**Comment by s4int** [comment]
This is just use of capabilities in my opinion even simpler than for sauce.

Testing this can be quite tricky because it requires working selenium grid.
You can think about unifying use of capabilities for grid and sauce, then tests can verify passing values.

**Comment by kahunacohen** [comment]
:+1:  Looks good to me. @hellmanj , @frishberg ?

File: robotpageobjects/page.py [hunk 1]
```diff
@@ -154,7 +155,13 @@ def __init__(self):
 
         self._attempt_sauce = self._validate_sauce_options()
 
-        # There's only a session ID when using a remote webdriver (Sauce, for example)
+        self._Capabilities = getattr(webdriver.DesiredCapabilities, self.browser.upper())
+        for grid_cap in self._Capabilities:
```
Comment (hellmanj) [inline robotpageobjects/page.py]: Seems like we're checking all options in `DesiredCapabilities` here not just grid-related ones.  I think that's fine, but  we shouldn't use "grid" in these variable names.

**Comment by hellmanj** [comment]
Also, it would be great if you could you update the readme with a brief section on using grid.

**Comment by s4int** [comment]
Ok, I will add short description how to use capabilities.

**Comment by kahunacohen** [comment]
Please also briefly mention in the README why choose sauce over grid and vice versa. We are pretty unfamiliar with grid.

**Comment by s4int** [comment]
Basically Sauce Labs is a fancy selenium grid. Grid is simpler and you can run it in your env.

**Comment by kahunacohen** [comment]
Looks good to me. In the read me we may want to specify these are options related to grid and not necessary for running in sauce. Otherwise looks good. Jared? Daniel?

**Comment by kahunacohen** [comment]
@hellmanj , @frishberg Are we ok to merge this? Do the option names make sense, or should they be pre-fixed with grid_?

**Comment by hellmanj** [comment]
It's probably fine for now, we can change later if it's apparent there is any confusion.

**Comment by kahunacohen** [comment]
OK I am going to reorder the options in the README and merge.

**Comment by kahunacohen** [comment]
I closed this because I pushed this forked branch to origin. I will open another PR there.

**Pull request closed without merging**